Packs/CheckpointFirewall/ReleaseNotes/2_1_5.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### CheckPoint Firewall v2
+- Improved login mechanism: restoration of session id is followed by a connection test, and failure to connect triggers a login.
```

Comment:
This message isn't clear for the average user, as the word `restoration` wouldn't be understood unless you understand the full context of how the integration works.
I'd say that the keyword should be `saved`, so it'll be something along the line of
```
usage of previously saved session id...
```